### ISSUE TITLE
Allow secondary color for vehicles

### ISSUE THREAD

**Issue by leoluecken** [eclipse/sumo#5395]
Just an idea for extending means of visualization.

**Comment by namdre** [comment]
is this obsolete with #5396 ?

**Comment by leoluecken** [comment]
No. Two colors means that the vehicle can be drawn in two different color, e.g. one for the roof and another for the rest of the chassis.
